setup.cfg

```diff
@@ -1,6 +1,3 @@
-[wheel]
```

Comment:
might be better

```
[bdist_wheel]
python-tag = py36.py37.py38.py39
```

Reply:
That would rename the built wheel from
`streamlink-1.6.0+31.g4cd86ba-py3-none-any.whl`
to
`streamlink-1.6.0+31.g4cd86ba-py36.py37.py38.py39-none-any.whl`

On the universal wheel, the python tag was just `py2.py3` and didn't include specific py3 versions:
`streamlink-1.6.0-py2.py3-none-any.whl`